src/Aspire.Hosting.Azure/Provisioning/Provisioners/AzureProvisioner.cs

```diff
@@ -104,11 +98,21 @@ async Task AfterProvisionAsync(IAzureResource resource)
             {
                 await resource.ProvisioningTaskCompletionSource!.Task.ConfigureAwait(false);
 
-                await SetStateAsync(resource, "Running").ConfigureAwait(false);
+                await UpdateStateAsync(resource, s => s with
+                {
+                    State = "Running",
+                    StateStyle = "success"
+                })
+                .ConfigureAwait(false);
             }
             catch (Exception)
             {
-                await SetStateAsync(resource, "FailedToStart").ConfigureAwait(false);
+                await UpdateStateAsync(resource, s => s with
+                {
+                    State = "Failed to Provision",
```

Comment:
later question: these likely need to be localized?

Reply:
Yep.